src/styles/main.scss

```diff
@@ -1,3 +1,72 @@
+  $day-size: 100px;
+  $next-day: $day-size + 1px;
+
 body {
   margin: 0;
 }
+
+.conteiner-calendar {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  padding-top: 30px;
+
+
+  .calendar {
```

Comment:
Would be great to create a separate file for calendar block

Reply:
> Would be great to create a separate file for calendar block

Comments from the previous reviewer aren't fixed